src/components/tabs/components/TabItem.vue

```diff
@@ -43,11 +44,18 @@ export default {
             type: String,
             required: true,
         },
+        /**
```

Comment:
Could we have a very brief summary here of what the BSH styling difference actually is, like in megalinks? It's captured in the story but it would be useful to be able to quickly tell what's going on, if we do have to have these temporary BSH versions through the component library